recipe/meta.yaml

```diff
@@ -22,7 +22,8 @@ requirements:
     - python >=3.7
     - grpcio >=1.38.1,<2.0dev
     - google-api-core-grpc >=1.34.0,<3.0.0dev,!=2.0.*,!=2.1.*,!=2.2.*,!=2.3.*,!=2.4.*,!=2.5.*,!=2.6.*,!=2.7.*,!=2.8.*,!=2.9.*,!=2.10.*
-    - proto-plus >=1.22.0,<2.0.0dev
+    # going with the tighter constrain for py3.11 to stay noarch: python
+    - proto-plus >=1.22.2,<2.0.0dev
```

Comment:
@tpdownes, @tswast and @hmaarrfk, we all agree that this is what we want, right?

Reply:
I think this is the sensible approach for now, regardless of what you Google folks decide to do in the next couple of days.

Reply:
Yes. This is what I'm suggesting

Reply:
LGTM